src/idsk/components/intro-block/template.njk

```diff
@@ -0,0 +1,86 @@
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M0.75708 13.103C0.75708 6.35398 6.36621 0.888672 13.2928 0.888672C20.2194 0.888672 25.8285 6.35398 25.8285 13.103C25.8285 15.4559 25.1301 17.7778 23.8094 19.7516L29.4827 25.2794C29.9551 25.7396 30.2392 26.3943 30.2392 27.0702C30.2392 28.464 29.058 29.6149 27.6276 29.6149C26.9347 29.6149 26.2611 29.3385 25.787 28.8584L20.1168 23.3497C18.0909 24.6367 15.7078 25.3172 13.2928 25.3172C6.36621 25.3172 0.75708 19.8519 0.75708 13.103ZM13.2928 1.68034C6.81494 1.68034 1.56958 6.7912 1.56958 13.103C1.56958 19.4147 6.81494 24.5256 13.2928 24.5256C15.6581 24.5256 17.9892 23.8275 19.9361 22.5143L20.2144 22.3265L26.3704 28.3071C26.6886 28.6308 27.1506 28.8232 27.6276 28.8232C28.6093 28.8232 29.4267 28.0267 29.4267 27.0702C29.4267 26.6046 29.2285 26.1513 28.9082 25.8392L22.7588 19.8475L22.9518 19.5759C24.2996 17.679 25.016 15.4076 25.016 13.103C25.016 6.7912 19.7706 1.68034 13.2928 1.68034ZM13.2928 5.97796C9.26151 5.97796 5.98029 9.17504 5.98029 13.103C5.98029 17.0309 9.26151 20.228 13.2928 20.228C17.3241 20.228 20.6053 17.0309 20.6053 13.103C20.6053 9.17504 17.3241 5.97796 13.2928 5.97796ZM5.16779 13.103C5.16779 8.73781 8.81278 5.18629 13.2928 5.18629C17.7728 5.18629 21.4178 8.73781 21.4178 13.103C21.4178 17.4681 17.7728 21.0196 13.2928 21.0196C8.81278 21.0196 5.16779 17.4681 5.16779 13.103Z" fill="white"/>
+                            </svg>
+                        </button>
+                    </div>
+                {% endif %}
+                <div>
+                    <ul class="idsk-intro-block-inline-list">
+                        <li class="idsk-intro-block-bottom-menu-caption govuk-caption-l">
+                            <a>Hľadáte toto?
+                            </a>
+                        </li>
+                        {% for bottomMenu in params.bottomMenu %}
+                            <li class="idsk-intro-block-inline-list-item">
+                                <a class="govuk-link idsk-intro-block-inline-list-item-a" href="{{bottomMenu.href}}">
+                                    {{ bottomMenu.item }}
+                                </a>
+                            </li>
+                        {% endfor %}
+                    </ul>
+                </div>
+            </div>
+            <div class="{{ 'idsk-intro-block-top-content-container-default' if params.classBackground=='app-pane-blue' or params.classBackground=='app-pane-grey'}}">
+                {% if params.classBackground=="app-pane-blue" %}
+                    <div id="idsk-intro-block-top-content-container-blue-background" class="govuk-grid-column-full govuk-grid-column-one-third-from-desktop app-pane-blue idsk-intro-block-text-colour">
+                        <h2>
+                            POPULÁRNY OBSAH
```

Comment:
pismo by malo byt male, upravit ho csskom ak je vobec treba... citacky to potom citaju inak